README.md

```diff
@@ -3,16 +3,27 @@ Example CNF
 
 [![PR Validation](https://github.com/openshift-kni/example-cnf/actions/workflows/push.yaml/badge.svg)](https://github.com/openshift-kni/example-cnf/actions/workflows/push.yaml)
 
-Example CNF is an OpenShift workload to exercice an SRIOV setup.
+Example CNF is an OpenShift workload to exercice an SRIOV setup, such as the following:
 
 ![Schema](documentation/schema.png)
 
 It is providing the following operators:
 
-* testpmd-operator
 * trex-operator
+    * It provides TRex Traffic Generator, decomposed in two components: a TRex server, deployed as `trexconfig-<x>` pod, which takes care of deploying and configuring a TRex server instance, and a TRex application, deplayed as `trex-app` job, that starts TRex server, generating traffic towards the system under test.
+    * In `trexconfig-<x>` logs, you can see the trex statistics printed periodically. The summary of the test execution can be seen at the end of the `trex-app` job logs.
+    * In our case, as we will see later on, the `trexconfig-<x>` pod has two interfaces connected to the same SRIOV network.
 * testpmd-lb-operator
+    * Its main component is a modified TestPMD instance, implementing a custom load balancing forwarding module, which is eventually used to perform load balancing between the ports of the deployed pod, called `loadbalancer-<x>`.
+    * This pod is composed by two containers: `loadbalancer`, which performs the load balancing forwarding, and `listener`, an auxiliary module that is listening to the CNFAppMac component created by the cnf-app-mac-operator to retrieve the MAC addresses of the TestPMD instances launched by the testpmd-operator (i.e. the CNF Applications), then serving this information to the `loadbalancer` container.
+    * To see the TestPMD statistics printed periodically for this module, you can rely on `loadbalancer` container logs.
+    * The `loadbalancer-<x>` pod has four network interfaces; in our case, two of them connected to the same SRIOV network than TRex, and the other two connected to the same SRIOV network than the CNF Application.
+    * The `loadbalancer-<x>` pod is not currently replicated; only one instance is deployed. It's supposed that, up to now, it's not belonging to any update-upgrade process of the example-cnf, so that it has to be deployed in an isolated worker node, whereas the other components of this CNF (TRex and CNF Application) have to be deployed in a different worker node.
+* testpmd-operator
+    * Final application, also known as CNF Application, which is a standard TestPMD instance using the default MAC forwarding module. Two replica pods are deployed, called `testpmd-app-<x>`, having each of them, in our case, two ports connected to the same SRIOV network, but different than the SRIOV network used by TRex.
+    * To see the TestPMD statistics printed periodically for this module, you can rely on `testpmd-app-<x>` container logs. Each log will offer you the statistics of each replica pod.
```

Comment:
When discussing an operator, we could probably discuss all the components, not just a pod. CR, pinning the state, is really the key here. Also, we deploy both loadbalancer and loadbalancer-lb, and you're discussing only loadbalancer-lb speaking about loadbalancer pod. That makes things pretty unclear

Reply:
honestly, I don't get your point about loadbalancer and loadbalancer-lb :sweat_smile:  there's just one loadbalancer and it's called loadbalancer. Maybe you're referring to TestPMD and TestPMD LB, I'll try to refactor that to make it clearer